### PR TITLE
Allows to use kafka topic names from clowder

### PR DESCRIPTION
**Pull request by petrblaho**
feat(kafka): read kafka topic names from clowder config

[EVNT-448]

### CODE REVIEW

**Comment by vkrizan** [comment]
Please cleanup properties:
* `kafka.return.topic`
* `kafka.ingress.topic`

File: splunk-quarkus/Dockerfile.jvm [hunk 1]
```diff
@@ -45,7 +45,7 @@ EXPOSE 8080
 USER 1001
 
 ENTRYPOINT ["sh", "-c", "java -Dquarkus.http.host=0.0.0.0 $JAVA_OPTIONS \
- -Dkafka.ingress.topic=$KAFKA_INGRESS_TOPIC\
+ -Dmp.messaging.kafka.ingress.topic=$KAFKA_INGRESS_TOPIC\
```
Comment (vkrizan) [inline splunk-quarkus/Dockerfile.jvm:48]: Let's remove it completely, the `-D` param and Docker `ENV`.

File: splunk-quarkus/clowdapp.yaml [hunk 2]
```diff
@@ -101,8 +96,6 @@ parameters:
     value: >-
       -Djava.util.logging.manager=org.jboss.logmanager.LogManager
       -Dquarkus.log.file.enable=false
-  - name: KAFKA_INGRESS_TOPIC
-    description: Kafka topic that the consumer takes messages from
     value: platform.notifications.tocamel
```
Comment (vkrizan) [inline splunk-quarkus/clowdapp.yaml:99]: You missed to remove this as well. It belonged to the removed param.

Comment (petrblaho) [reply splunk-quarkus/clowdapp.yaml:99]: oops :-)

**Review by vkrizan — APPROVED** [review]
LGTM

**Pull request merged**